aws-lc-rs/src/cipher.rs

```diff
@@ -141,75 +141,26 @@ pub(crate) mod aes;
 pub(crate) mod block;
 pub(crate) mod chacha;
 pub(crate) mod key;
+mod padded;
 
+pub use padded::{PaddedBlockDecryptingKey, PaddedBlockEncryptingKey, PaddingStrategy};
```

Comment:
Was making `PaddingStrategy` public intentional?  Just double checking since the PR description mentioned no changes to the public API

Reply:
Good catch. Will fix.